### ISSUE TITLE
Immich lacks accessibility for blind users

### ISSUE THREAD

**Issue by cyrmax** [immich-app/immich#10572]
### The bug

Neither Immich web interface, nor Immich mobile application for Android have enough accessibility for blind users.
Accessibility is important because Immich has automatic photo tagging, custom description field for each photo and facial recognition and that's why it could be very helpful for blind and visually impaired users.
But Immich UI works not well with screen reading software for blind users.

### The OS that Immich Server is running on

Not important in this case

### Version of Immich Server

v1.106.4

### Version of Immich Mobile App

v1.106.3

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Not important here
```

### Your .env content

```Shell
Not important here
```

### Reproduction steps

```bash
1. Install a screen reading software (NVDA for Windows or Google Talk Back for Android);
2. Wear your headphones and close your eyes (or turn off your display completely);
3. Try to use Immich normally by using only screen reader output and hints;
4. For example try to find a specific photo by its date, orientation any other non-visual info;
5. Try to change the photo info, to add a description, to manage other photo or video info;
6. For comparison try to use Google Photo for the same tasks without looking at your screen.
```

**Comment by bo0tzz** [comment]
@ben-basten you've already done quite a bit of work in this direction on the web client, right?

**Comment by ben-basten** [comment]
Hey @cyrmax, thanks for the feedback!

I've been focusing on improving the accessibility of the web interface recently, using Apple VoiceOver to test and comparing to Google Photos for reference. 

As a sighted developer, I also value feedback from immich users on Discord or GitHub to learn more about what the accessibility pain points are. Are there any particular features or tasks in immich mobile or web apps that you find challenging to use? Any insights that you share will be helpful for prioritizing future work!

I am currently working on #10336, to improve screen readers announcing photo descriptions in the mobile app.

**Comment by cyrmax** [comment]
Hello!
Thanks for so quick reply and even more for the work you are doing.
Unfortunately deep accessibility testing is not so quick and easy, so I cannot spend enough time to it until next weekend.
But I will find a time and try to write as much as I can.
Btw I didn't try an iOS app yet, but can check also.
I have an iPhone but it is only for work and I don't use it so often, so just didn't come to my mind.

**Comment by ben-basten** [comment]
> Unfortunately deep accessibility testing is not so quick and easy, so I cannot spend enough time to it until next weekend. But I will find a time and try to write as much as I can.

I know accessibility testing is time-intensive, especially testing for multiple platforms. Thanks in advance for any time that you are able to put into this, I appreciate your feedback!
